components/RichText/Quote.module.scss

```
@@ -4,13 +4,13 @@
     display: flex;
     justify-content: center;
     margin: $spacing-6 auto;
-    padding: $spacing-4;
-    background: $color-base-100;
+    padding-top: $spacing-4;
+    padding-bottom: $spacing-4;
```

You can use the short syntax here:
```suggestion
    padding: $spacing-4 0;
```